data/data/aws/vpc/sg-master.tf

```diff
@@ -176,6 +176,26 @@ resource "aws_security_group_rule" "master_ingress_kube_controller_manager_from_
   to_port   = 10252
 }
 
+resource "aws_security_group_rule" "master_ingress_crio_metrics" {
+  type              = "ingress"
+  security_group_id = "${aws_security_group.master.id}"
+
+  protocol  = "tcp"
+  from_port = 9537
```

Comment:
This should fall into the default metrics range of 9000-9999 which we added in 3248996de231 (#683).  Maybe that should be open on compute nodes as well as masters?  CC @smarterclayton

Reply:
My understanding is that from the worker prometheus is on we can't scrape metrics on other workers /cc @brancz

Reply:
Yes, you can if you're in 9000-9999 (which is why we require it.

Reply:
Yes I don't think we need this for the master, just for the workers. That seems to be in line with my findings from here: https://github.com/openshift/cluster-monitoring-operator/pull/228#issuecomment-461198657

Reply:
I'll drop the lines from masters thanks